module/dialogs/CharacterBurnerDialog.ts

```diff
@@ -404,7 +404,7 @@ export class CharacterBurnerDialog extends Application {
                 // if a skill is already included don't double include it
                 if (!$(this.element).find(`select[name="skillName"]`).filter((_, element) => $(element).val() === itemName).length) {
                     // Set the value, creating a new option if necessary
-                    if (element.find("option[value='" + itemName + "']").length) {
+                    if (element.find("option[value='" + escapeQuotes(itemName) + "']").length) {
```

Comment:
Maybe a good change here, since the code is being touched, would be to use string interpolation
```suggestion
                    if (element.find(`option[value='${escapeQuotes(itemName)}']`).length) {
```

Reply:
ditto the rest of the places where `escapeQuotes` was added.

Reply:
I'll get on that.